pom.xml

```diff
@@ -83,7 +83,7 @@
 
   <modules>
     <module>planetiler-core</module>
-    <module>planetiler-basemap</module>
```

Comment:
Also can you add `planetiler-openmaptiles` to the `sonar.exclusions` property up above? It would make more sense for sonar to run against that code in the `planetiler-openmaptiles` repo itself, and have planetiler only check its own code.

Reply:
`planetiler-openmaptiles` moved to `sonar.exclusions`